third_party/move/move-core/types/src/function.rs

```diff
@@ -407,11 +429,7 @@ impl fmt::Display for MoveClosure {
             captured,
         } = self;
         let captured_str = mask
-            .merge_placeholder_strings(
-                mask.captured_count() as usize,
-                captured.iter().map(|v| v.1.to_string()).collect(),
-            )
-            .unwrap_or_else(|| vec!["*invalid*".to_string()])
+            .merge_placeholder_strings(captured.iter().map(|v| v.1.to_string()).collect())
             .join(",");
```

Comment:
```suggestion
            .join(", ");
```

Reply:
I believe other "lists" are displayed as `a, b, c` and not `a,b,c`, so we might want to be consistent.

Reply:
Also forgot ;-)

Reply:
Got it, added spaces then 👍